README.markdown

```diff
@@ -4,22 +4,22 @@ Enables instructors to create questions with free-text responses.
 ## TODO List:
```

Comment:
It would be helpful to include some instructions for how to set this up in Studio e.g. what do I put in the Advanced Modules list.